pkg/app/command/config.go

```diff
@@ -0,0 +1,27 @@
+package command
+
+import (
+	"github.com/pkg/errors"
+	"github.com/spf13/afero"
+	"github.com/spf13/viper"
```

Comment:
**[errcheck]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
 could not import github.com/spf13/viper (cannot find package "github.com/spf13/viper" in any of: